__tests__/hooks/useGroupInfo.test.js

```diff
@@ -186,6 +192,7 @@ describe('useGroupInfo should run correctly', () => {
     ]
   }));
 
+  const mockQueryMemberError = jest.fn().mockRejectedValue('error');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new mock function `mockQueryMemberError` has been introduced which rejects with an error. It's important to ensure that your code can handle errors and edge cases correctly, so adding tests for these scenarios is a good practice.
```diff
+ const mockQueryMemberError = jest.fn().mockRejectedValue('error');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->